app/lib/features/activities/providers/session_providers.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:acter/features/home/providers/client_providers.dart';
+import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart' show DeviceRecord;
+import 'package:flutter/foundation.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+final allSessionsProvider = FutureProvider<List<DeviceRecord>>(
+  (ref) async {
+    final client = ref.watch(clientProvider)!;
+    final manager = client.sessionManager();
+    final sessions = (await manager.allSessions()).toList();
+    debugPrint('$sessions');
+    return sessions;
+  },
+);
+
+final verifiedSessionsProvider = FutureProvider<List<DeviceRecord>>(
```

Comment:
can't we collapse all these other sessions providers into `ref.watch(allSessionProvider)` that then filters them upon some criteria rather than have separate backend functions without much actual content (and thus more memory overhead), please?!?

Reply:
Fixed by https://github.com/acterglobal/a3/pull/852/commits/e61a9a15674d4c8accc839b5a425492fe9d1f65f